docs/devenv_rhel8.md

```diff
@@ -0,0 +1,248 @@
+
+Click on Begin Installation and wait until the OS installation is complete.
+
+### Configuring VM
+Log into the virtual machine using SSH with the `microshift` user credentials.
+
+Run the following commands to upgrade the system, install basic dependencies, enable remote Cockpit console and configure SUDO.
+```bash
+sudo dnf update -y
+sudo dnf install -y git cockpit make golang
+sudo systemctl enable --now cockpit.socket
+echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee -a /etc/sudoers
+```
+You should now be able to access the VM Cockpit console using `https://<vm_ip>:9090` URL.
+
+## Build MicroShift 
+Log into the development virtual machine with the `microshift` user credentials.
+Clone the repository to be used for building various artifacts.
+```bash
+git clone https://github.com/redhat-et/microshift.git
+cd microshift
+```
+
+### Executable
+Run `make` command in the top-level directory. If necessary, add `DEBUG=true` argument to the `make` command for building a binary with debug symbols.
+```bash
+make
+```
+The artifact of the build is the `microshift` executable file located in the top level directory of the source tree.
+
+### RPM Packages
+Run make command with the `rpm` or `srpm` argument in the top-level directory. 
+```bash
+make rpm
+make srpm
+```
+
+The artifacts of the build are located in the `packaging` directory.
+```bash
+$ find packaging -name \*.rpm
+packaging/rpm/_rpmbuild/RPMS/x86_64/microshift-4.10.0-nightly_1654189204_34_gc871db21.el8.x86_64.rpm
+packaging/rpm/_rpmbuild/RPMS/noarch/microshift-selinux-4.10.0-nightly_1654189204_34_gc871db21.el8.noarch.rpm
+packaging/rpm/_rpmbuild/SRPMS/microshift-4.10.0-nightly_1654189204_34_gc871db21.el8.src.rpm
+```
+
+## Run MicroShift Executable
+Log into the development virtual machine with the `microshift` user credentials.
+### Installing Clients
+Run the following commands to install `oc` and `kubectl` utilities.
+```bash
+curl -O https://mirror.openshift.com/pub/openshift-v4/$(uname -m)/clients/ocp/stable/openshift-client-linux.tar.gz
+sudo tar -xf openshift-client-linux.tar.gz -C /usr/local/bin oc kubectl
+rm -f openshift-client-linux.tar.gz
+```
+### Runtime Prerequisites
+Run the following commands to install CRI-O.
+```bash
+sudo subscription-manager repos --enable rhocp-4.10-for-rhel-8-$(uname -i)-rpms
+sudo dnf install -y cri-o cri-tools
+sudo systemctl enable crio --now
+```
+### Running MicroShift
+Run the MicroShift in the background using the following command.
+```bash
+nohup sudo ./microshift run >> /tmp/microshift.log &
+```
+Examine the `/tmp/microshift.log` log file to ensure successful startup.
+
+Copy `kubeconfig` to the default location that can be accessed without the administrator privilege.
+```bash
+mkdir ~/.kube
+sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig > ~/.kube/config
+```
+
+Verify that the MicroShift is running.
+```bash
+oc status
```

Comment:
It's hard to get a clear overview of MicroShift status. I think `oc status` is good for OpenShift environments, but not so good for MicroShift.

wdyt about using `oc get cs`? (although it's gonna be deprecated soon)

```
oc get cs
W0603 12:41:06.770759   45461 warnings.go:67] v1 ComponentStatus is deprecated in v1.19+
NAME                 STATUS    MESSAGE                         ERROR
scheduler            Healthy   ok                              
controller-manager   Healthy   ok                              
etcd-0               Healthy   {"health":"true","reason":""}   
```
or `oc get all -A`

Just a suggestion.

Reply:
Changed as per your suggestion.